plugins/base/src/test/kotlin/model/PropertyTest.kt

```diff
@@ -152,12 +155,15 @@ class PropertyTest : AbstractModelTest("/src/main/kotlin/property/Test.kt", "pro
         ) {
             with((this / "property").cast<DPackage>()) {
                 with((this / "Bar" / "property").cast<DProperty>()) {
-                    dri.classNames equals "Foo"
+                    dri.classNames equals "Bar"
                     name equals "property"
                     children counts 0
                     with(getter.assertNotNull("Getter")) {
                         type.name equals "Int"
                     }
+                    extra[InheritedMember]?.inheritedFrom?.values?.single()?.run {
+                        classNames equals "Foo"
+                    }
```

Comment:
I believe this test was wrong, it was correct in the context of previous implementation, but we expect property to have its class in the DRI and actual class it is coming from in the InheritedMember extra property, am I right?